### PR TITLE
Remove unneeded dependencies

### PR DESCRIPTION
**Pull request by ZakariaRidouh**
This helps reduce the size of the App (my Butter directories are massive)

### CODE REVIEW

**Comment by vankasteelj** [comment]
that's a good idea, but isnt gitlab still defined in about?

**Comment by xaiki** [comment]
again, string changes in the mix, aren't those modules used anywhere ? their functionalities should be removed at the same time.

File: src/app/templates/about.tpl [hunk 1]
```diff
@@ -19,16 +19,16 @@
         <div class="text-about">
             <div class="full-text">
                 <%= i18n.__("%s is the result of many developers and designers putting a bunch of APIs together to make the experience of watching torrent movies as simple as possible.", Settings.projectName) %><br/>
-                <%= i18n.__("We are an open source project. We are from all over the world. We love our movies. And boy, do we love popcorn.") %>
+                <%= i18n.__("We are an open source project. We are from all over the world. We love our movies. And boy, do we love our Popcorn.") %>
```
Comment (vankasteelj) [inline src/app/templates/about.tpl]: nuuuuuuuuh! you broke internationalization

**Comment by xaiki** [comment]
i'm pretty sure you're breaking stuff.

```sh
git grep strike src/app/lib
src/app/lib/views/torrent_collection.js:                var strike = require('strike-api');
src/app/lib/views/torrent_collection.js:                strike.search(input, category).then(function (result) {
```
```sh
git grep gitlab src/app/lib
src/app/lib/views/issue.js:    var BT_id = 13, //ID of project (got with gitlab.projects.all). 13 is for 'butterproject/butter'
src/app/lib/views/issue.js:            if (AdvSettings.get('gitlabPassword') && AdvSettings.get('gitlabMail')) {
src/app/lib/views/issue.js:                $('#issue-email').val(AdvSettings.get('gitlabMail')),
src/app/lib/views/issue.js:                    $('#issue-pw').val(AdvSettings.get('gitlabPassword'));
src/app/lib/views/issue.js:            var gitlab = require('gitlab')({
src/app/lib/views/issue.js:            gitlab.projects.issues.list(BT_id, {
src/app/lib/views/issue.js:            var gitlab = require('gitlab')({
src/app/lib/views/issue.js:            gitlab.issues.create(
src/app/lib/views/issue.js:                    AdvSettings.set('gitlabMail', $('#issue-email').val());
src/app/lib/views/issue.js:                    AdvSettings.set('gitlabPassword', $('#issue-pw').val());
src/app/lib/views/issue.js:            var gitlab = require('gitlab')({
src/app/lib/views/issue.js:            gitlab.users.session(email, password, function (response) {
```

File: src/app/templates/about.tpl (hunk 1 above)
Comment (kiefferbp) [inline src/app/templates/about.tpl]: I prefer the old text. I'm not sure how this line is supposed to be an improvement.

**Pull request closed without merging**